src/test/regress/expected/isolation_multi_shard_modify_vs_all.out

```diff
@@ -0,0 +1,355 @@
+Parsed test spec with 2 sessions
+
+starting permutation: s1-begin s1-update_all_value_1 s2-begin s2-select s1-commit s2-select s2-commit
+step s1-begin: 
+    BEGIN;
+
+step s1-update_all_value_1: 
+	UPDATE users_test_table SET value_1 = 3;
+
+step s2-begin: 
+	BEGIN;
+
+step s2-select: 
+	SELECT * FROM users_test_table ORDER BY value_1;
+
+user_id        value_1        value_2        value_3        
+
+1              5              6              7              
+2              12             7              18             
+8              18             13             4              
+6              21             11             25             
+3              23             8              25             
+7              27             12             18             
+5              35             10             17             
+4              42             9              23             
+step s1-commit: 
+    COMMIT;
+
+step s2-select: 
+	SELECT * FROM users_test_table ORDER BY value_1;
+
+user_id        value_1        value_2        value_3        
+
+1              3              6              7              
+5              3              10             17             
+8              3              13             4              
+3              3              8              25             
+4              3              9              23             
+7              3              12             18             
+6              3              11             25             
+2              3              7              18             
+step s2-commit: 
+	COMMIT;
+
+restore_isolation_tester_func
+
+               
+
+starting permutation: s1-begin s1-update_all_value_1 s2-begin s2-update_all_value_1 s1-commit s2-commit
+step s1-begin: 
+    BEGIN;
+
+step s1-update_all_value_1: 
+	UPDATE users_test_table SET value_1 = 3;
+
+step s2-begin: 
+	BEGIN;
+
+step s2-update_all_value_1: 
+	UPDATE users_test_table SET value_1 = 6;
+ <waiting ...>
+step s1-commit: 
+    COMMIT;
+
+step s2-update_all_value_1: <... completed>
+step s2-commit: 
+	COMMIT;
+
+restore_isolation_tester_func
+
+               
+
+starting permutation: s1-begin s1-update_value_1_of_1_or_3 s2-begin s2-update_value_1_of_5_or_7 s1-commit s2-commit s2-select
+step s1-begin: 
+    BEGIN;
+
+step s1-update_value_1_of_1_or_3: 
+	UPDATE users_test_table SET value_1 = 5 WHERE user_id = 1 or user_id = 3;
+
+step s2-begin: 
+	BEGIN;
+
+step s2-update_value_1_of_5_or_7: 
+	UPDATE users_test_table SET value_1 = 4 WHERE user_id = 5 or user_id = 7;
+ <waiting ...>
```

Comment:
this `<waiting ...>` is slightly surprising since they don't hit the same user IDs. I guess they have a shard in common?